app/serializers/cache/additional_code_serializer.rb

```diff
@@ -34,7 +34,10 @@ def as_json
             validity_end_date: measure.validity_end_date,
             goods_nomenclature_item_id: measure.goods_nomenclature_item_id,
             goods_nomenclature_sid: measure.goods_nomenclature_sid,
+            goods_nomenclature_id: measure.goods_nomenclature_sid,
```

Comment:
Do we need it?
also, I am wondering why you added the`goods_nomenclature_id` here, but not in the other serializers.

Reply:
Good catch